geeqie-no-changelog.patch

```diff
@@ -0,0 +1,14 @@
+--- geeqie-1.4.orig/Makefile.am	2017-12-31 07:31:21.000000000 -0500
++++ geeqie-1.4/Makefile.am	2018-01-01 15:05:58.742068166 -0500
+@@ -10,9 +10,9 @@
+ readmedir = @readmedir@
+ 
```

Comment:
I don't see why this patch is needed? if the file isn't desired, it can be removed with a cleanup.

Reply:
Without this patch, I get this error message:
```
fatal: Invalid revision range 1b58572cf58e9d2d4a0305108395dab5c66d3a09..HEAD
tail: cannot open 'ChangeLog' for reading: No such file or directory
mv: cannot stat 'ChangeLog.815.new.html': No such file or directory
 /usr/bin/mkdir -p '/app/share/doc/geeqie-a55c930'
 /usr/bin/install -c -m 644 README.md COPYING ./ChangeLog TODO README.lirc AUTHORS ./ChangeLog.html '/app/share/doc/geeqie-a55c930'
/usr/bin/install: cannot stat './ChangeLog': No such file or directory
/usr/bin/install: cannot stat './ChangeLog.html': No such file or directory
make[2]: *** [Makefile:654: install-readmeDATA] Error 1
make[2]: Leaving directory '/run/build/geeqie'
make[1]: *** [Makefile:987: install-am] Error 2
make[1]: Leaving directory '/run/build/geeqie'
make: *** [Makefile:681: install-recursive] Error 1
Error: module geeqie: Der Kindprozess wurde mit Status 2 beendet

```

Reply:
Right